enterprise-4.13 OADP-4657: Release Notes for 1.4.0 for OCP 4.13 (#80585)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc

:_mod-docs-content-type: ASSEMBLY
[id="oadp-1-4-release-notes"]
= OADP 1.4 release notes
include::_attributes/common-attributes.adoc[]
include::_attributes/attributes-openshift-dedicated.adoc[]
:context: oadp-1-4-release-notes

toc::[]

The release notes for {oadp-first} describe new features and enhancements, deprecated features, product recommendations, known issues, and resolved issues.

include::modules/oadp-1-4-0-release-notes.adoc[leveloffset=+1]
include::modules/oadp-backing-up-dpa-configuration-1-4-0.adoc[leveloffset=+3]
include::modules/oadp-upgrading-oadp-operator-1-4-0.adoc[leveloffset=+3]
[role="_additional-resources"]
.Additional resources

> * xref:../../../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Updating installed Operators]

Reviewer: ```suggestion
* xref:../../../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Changing the update channel for an Operator]
```

[id="oadp-converting-dpa-to-new-version-1-4-0_{context}"]
=== Converting DPA to the new version

> To upgrade from OADP 1.3 to 1.4, no Data Protection Application (DPA) changes are required.

Reviewer 2: To keep the customer action center here, adding the suggested verbiage is needed: https://redhat-documentation.github.io/supplementary-style-guide/#_principle_1_customer_focus_and_action_orientation
